static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -475,17 +469,20 @@ class Actions extends Component<Props, State> {
                   priority: 'danger',
                   label: t('Delete'),
                   hidden: !hasAccess,
-                  disabled: !getIssueCapability(group.issueCategory, 'delete').enabled,
-                  onAction: () => this.openDeleteModal(),
+                  disabled: !deleteCap.enabled,
+                  tooltip: deleteCap.disabledReason,
+                  onAction: deleteCap.enabled ? () => this.openDeleteModal() : undefined,
```

Comment:
massive nit: we should use `null` for things we're setting ourselves 😄

Reply:
won't matter once `onAction` is fixed to take `disabled` into account tho

Reply:
I don't think the types will allow for it, and I don't think they should allow for it :)

Setting this as undefined is effectively leaving it as unset.

Reply:
That's a good point, agreed!